### PR TITLE
feat(proto): improve ergonomics for unwrapping types

### PR DESCRIPTION
**Pull request by joshuachp**
Since all the enum (oneof) types are wrapped by an Option, the deep nesting makes the pattern matching became cumbersome.

I've added a different method for each variant, which will return an option of the inner type.

### CODE REVIEW

File: src/data/astarte_handler.rs [hunk 1]
```diff
@@ -294,6 +294,7 @@ mod test {
     use crate::data::astarte::{AstartePublisher, AstarteRunner, AstarteSubscriber};
     use crate::data::mock_astarte_sdk::MockAstarteDeviceSdk;
     use crate::error::AstarteMessageHubError;
+    use crate::proto_message_hub::astarte_data_type_individual::IndividualData;
```
[the remark this answers is not in the document]
Comment (joshuachp) [reply src/data/astarte_handler.rs]: Done

File: src/lib.rs [hunk 2]
```diff
@@ -30,5 +30,248 @@ mod types;
 
 #[allow(clippy::all)]
 pub mod proto_message_hub {
+    use self::{astarte_data_type::Data, astarte_message::Payload};
```
[the remark this answers is not in the document]
Comment (joshuachp) [reply src/lib.rs]: Yep, I was thinking the same. Done in [2ced3be](https://github.com/astarte-platform/astarte-message-hub/pull/88/commits/2ced3beffbf5e80a544805eede59e160dc279d36)

**Comment by harlem88** [comment]
For the new functions introduced in the `proto_message_hub.rs` file, the adding of a docstring would make them easier to understand and also explain why they were introduced.

**Comment by joshuachp** [comment]
> For the new functions introduced in the `proto_message_hub.rs` file, the adding of a docstring would make them easier to understand.

Done 🚀

**Review by harlem88 — APPROVED** [review]

**Pull request merged**